compiler/test/formatter_outputs/binops.gr

```diff
@@ -11,3 +11,25 @@ if (
   bafdfdfdfdfdfdddefdfdfdfdfdr &&
   badfdfdfdfdfdfdffdffdfdfdz
 ) true
+
+let a = (2 >> 1) << 1
+
+let a1 = ((2 >> 1) << 1) >> 1
+
+let x = 1 * (2 + 3)
+let y = 1 * 2 + 3
+let cond = false && false || false
+
+let z = 1 + 2 + 3
+
+let p = 1 + 2 - 3
+
+let p2 = 1 - 2 + 3
+
+let y = 1 / 2 + 3
+
+let ya = 1 / (2 + 3)
+
+let y1 = 1 + 2 /3
```

Comment:
extra space in here?

Reply:
Oh that's because constants are formatted as-is and it's a rational literal. I'll open an issue about that separately.

Reply:
https://github.com/grain-lang/grain/issues/1516